oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/metrics/Metrics.java

```diff
@@ -102,7 +102,7 @@ public long toTimeBucketInHour() {
         if (isMinuteBucket()) {
             return timeBucket / 100;
         } else {
-            throw new IllegalStateException("Current time bucket is not in minute dimensionality");
+            throw new IllegalStateException("Current time bucket is not in minute dimensionality: " + timeBucket);
```

Comment:
I think you could see the time bucket from index-0? It should mean no value of time bucket, right?

Reply:
Is this log really helpful?